artifacts.html

```diff
@@ -0,0 +1,93 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="utf-8">
+  <meta name="Description" content="INF 7420 Project">
+  <meta name="viewport" content="width=device-width, initial-scale=1">
+  <title>Artifacts Page</title>
+</head>
+<body>
+<!-- --------------------------------HEADER CONTENT--------------------------------------------------------------------------------- -->
+<header>
+<!-- Social Media Images -->
+
+<a href="https://www.facebook.com">
```

Comment:
These list of links to social media can be written as an unordered list.

Reply:
Changes have been made, please review and comment if there's anything additional to be done for this.